cdci_osa_plugin/exposer.py

```diff
@@ -35,10 +35,13 @@
 
 # Project
 # relative import eg: from .mod import f
-
-
 from .osa_isgri import  osa_isgri_factory
 from .osa_jemx import osa_jemx_factory
 from .osa_fake import osa_fake_factory
 
 instr_factory_list = [osa_isgri_factory, osa_jemx_factory, osa_fake_factory]
+instr_name_dict = dict(
+    isgri=osa_isgri_factory(),
```

Comment:
isn't calling `osa_isgri_factory()` ending up calling instrument constructor?

Reply:
ops, setting wrongly the entry